geometry_msgs/Pose[] pose
float64[] info_gain